express-get/server.ts

```diff
@@ -0,0 +1,49 @@
+import express from 'express';
+import pg from 'pg';
+import { errorMiddleware } from './lib/index.js';
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/pagila',
+  ssl: {
+    // Allow non-SSL traffic to localhost
+    rejectUnauthorized: false,
+  },
+});
+
+const app = express();
+
+app.get('/api/films', async (req, res, next) => {
+  try {
+    const sql = `
+    select * from "films"
+    `;
+    const result = await db.query(sql);
+    const films = result.rows;
+    res.json(films);
+  } catch (err) {
+    next(err);
+  }
+});
+
+app.get('/api/films/:filmId', async (req, res, next) => {
+  try {
+    const sql = `
+    select "filmId" from "films"
+    `;
+    const result = await db.query(sql);
+    const films = result.rows;
```

Comment:
You are returning an array here instead of a single object.

You also need to check that you actually retrieved a film, and return a 404 if there is not one.